Ignore-Files-Committed-To-Git-Repo.md

```diff
@@ -0,0 +1,21 @@
+# Untrack files previously committed from new .gitignore
+
+To untrack a <i>single</i> file, ie stop tracking the file but not delete it from the system use:
```

Comment:
Use [markdown](https://guides.github.com/features/mastering-markdown) instead.
`<i>single<i>` to `_single_`